lib/sparse/src/index/search_context.rs

```diff
@@ -17,28 +18,29 @@ pub struct IndexedPostingListIterator<'a> {
     query_weight: DimWeight,
 }
 
-/// Making this larger makes the search faster but uses more memory
-const ADVANCE_BATCH_SIZE: usize = 1_000;
+/// Making this larger makes the search faster but uses more (pooled) memory
+const ADVANCE_BATCH_SIZE: usize = 10_000;
```

Comment:
increasing the batch size

Reply:
8192 ?

Reply:
I'd rather not go under 10k as the performance seems to respond well to the value.

If you really want a power of 2 I'd vote for `16_384` :)

Reply:
I will bench it quickly :scientist:

Reply:
Results of the bench:

- 16k performs overall worst than 10k
- 8k has a significant larger max value than 10k

Decision: I will stick to 10k